services/m3aggregator/config/aggregator.go

```diff
@@ -385,3 +529,15 @@ func setMetricPrefixOrSuffix(
 	}
 	return fn([]byte(str))
 }
+
+func instanceID(address string) (string, error) {
```

Comment:
Currently the address is "0.0.0.0:1234" or "127.0.0.1:1234", there is no hostname in it.